src/Resources/views/Admin/Form/_scripts.html.twig

```diff
@@ -1,237 +1,6 @@
-{% include '@SyliusUi/_javascripts.html.twig' with {'path': 'https://jkbrzt.github.io/rrule/dist/es5/rrule.min.js'} %}
+<script src="{{ asset('admin-click-n-collect.js', 'click-n-collect.assets.package') }}"></script>
 
 <script type="text/javascript">
 // Wait DOM initialization
-document.addEventListener('DOMContentLoaded', function() {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Are you sure you want to remove this code ?

Reply:
yes this code is moved in `admin-click-n-collect.`